src/components/VideoPlayer.vue

```diff
@@ -273,7 +273,7 @@ export default {
 
             const lbry = this.getPreferenceBoolean("disableLBRY", false)
                 ? null
-                : this.video.videoStreams.filter(stream => stream.quality === "LBRY")[0];
+                : this.video.videoStreams.filter(stream => stream.quality.indexOf("LBRY") !== -1)[0];
```

Comment:
Just a question, why not using `stream.quality.includes("LBRY")` instead?

Reply:
Didn't know such a thing existed lol, I've always looked for a "contains" method, thanks!